source/guide_bitwarden.rst

```diff
@@ -288,6 +288,7 @@ Updating bitwarden_rs is really easy. Just stop the server, pull everything and
  [isabell@stardust ~]$ cd ~/vaultwarden
  [isabell@stardust bitwarden_rs]$ supervisorctl stop vaultwarden
  [isabell@stardust bitwarden_rs]$ git pull
+ [isabell@stardust bitwarden_rs]$ git checkout origin/async
```

Comment:
I wasn't sure what to checkout here. The guide says in line 66 to use `origin/async` because of the missing dependency updates. `origin/async` didn't get any new updates so far, though, so I suppose the whole `Update` section isn't needed until the dependencies are updated and we can use the current version tags.